README.md

```diff
@@ -39,3 +41,47 @@ GooglePlayInstant;` statements will have to be updated.
 
 1.  Change any `using GooglePlayInstant;` statements to `using
     Google.Play.Instant;`.
+
+## Known issues
+
+### Launching an instant app from the Play Store redirects to the browser
+
+This issue occurs when launching an instant app if the following is true:
+1. Your currently released instant app is launchable via a custom URL.
+1. You have uploaded a version of your instant app that does not specify a custom  to alpha or internal test.
+1. You are trying to launch the non-production version of your instant app.
+
+The latest plugin no longer supports the ability to set a custom instant apps URL. If your app previously included a custom instant apps URL, uploading an app built with the latest plugin could trigger this issue.
+
+There are two workarounds:
+1. The issue will not occur for an instant app in production, so release the app to production to make the issue go away.
+2. If you'd prefer to fix the issue in alpha, add the browsable intent filter and default url tags to the UnityPlayerActivity in your app's manifest:
+```xml
+            <intent-filter
```

Comment:
If it's not too much of a pain, reduce or remove the indent on all of these lines